Week03/lesson06/homework/startup_matchmaker/css/style.css

```diff
@@ -0,0 +1,224 @@
+  line-height: 0;
+  height: 0;
+}
+
+.clear {
+ clear: both;
+}
+
+/******************************************
+/* BASE STYLES
+/*******************************************/
+
+body {
+  line-height: 1.5;
+  font-family: arial, helvetica, sans-serif;
+  font-size: 18px;
+}
+
+/******************************************
+/* LAYOUT
+/*******************************************/
+
+/* Center the container */
+.container {
+  width: 960px;
+  margin: 0 auto;
+}
+
+header {
+  height: 100px;
+  background-color: #000;
+  border-bottom: 6px solid yellow;
+}
+
+h1 {
+  font-size: 30px;
+  color: #fff;
+  display: inline-block;
+  padding-top: 25px;
+}
+
+nav a {
+  display:inline-block;
+  height: 94px;
+  padding-top: 35px;
+  padding-left: 10px;
+  padding-right: 10px;
+  text-decoration: none;
+  color: #fff;
+  vertical-align: top;
+}
+
+
+nav {
+  float: right;
+  vertical-align: top;
+}
+
+nav a:hover {
+  color: yellow;
+}
+
+.banner {
+  text-align: center;
+  height: 100px;
+  background-color: #d5d9d3;
+  padding-top: 35px;
+}
+
+.heroImage {
+  height: 600px;
+  background: url(../images/coworking.jpg) no-repeat center bottom;
+  background-size: cover;
+  -webkit-box-shadow: inset 0px 5px 5px 0px rgba(0,0,0,0.52);
+  -moz-box-shadow: inset 0px 5px 5px 0px rgba(0,0,0,0.52);
+  box-shadow: inset 0px 5px 5px 0px rgba(0,0,0,0.52);
+}
+
+.heroImage .container {
+  padding-top: 80px;
+}
+
+.column {
+  width: 33.3%;
+  float: left;
+  padding: 30px;
+  border-right: 1px solid #b7b5ba;
+  margin: 30px 0;
+}
+
+.column:last-child {
+  border: none;
+}
+
+.column.cta {
+  float: right;
+  margin: none;
+}
+
+footer {
+  height: 75px;
+  background-color: #000;
+  padding-top: 30px;
+}
+
+
+
+/******************************************
+/* OTHER STYLES
+/*******************************************/
+
+h1, h2, h4 {
+  font-family: georgia, serif;
+  font-weight: bold;
+}
+
+h3 {
+  font-size: 24px;
+  font-weight: bold;
+}
+
+h3, h4 {
+  margin-bottom: 20px;
+}
+
+h2 {
+  font-style: italic;
+  font-size: 24px;
+}
+
+h4 {
+  font-size: 20px;
+}
+
+p {
+  margin-bottom: 20px;
+}
+
+.heroImage .column {
+  background: rgba(255, 255, 255, 0.5);
+  border: 1px solid #666;
+  -webkit-box-shadow: 0px 0px 5px 2px rgba(0,0,0,0.25);
+  -moz-box-shadow: 0px 0px 5px 2px rgba(0,0,0,0.25);
+  box-shadow: 0px 0px 5px 2px rgba(0,0,0,0.25);
+}
+
+
+.boldLink, .button {
+  font-family: oswald, sans-serif;
+  font-weight: 700;
+  letter-spacing: 2px;
+  text-transform: uppercase;
+}
+
+
+.button {
+  display: block;
+  -webkit-border-radius: 5;
+  -moz-border-radius: 5;
+  border-radius: 5px;
+  -webkit-box-shadow: 0px 3px 0px #666666;
+  -moz-box-shadow: 0px 3px 0px #666666;
+  box-shadow: 0px 3px 0px #666666;
+  color: #000000;
+  font-size: 20px;
+  background: #b1b2b3;
+  padding: 15px 0 15px 0;
+  text-decoration: none;
+  text-align: center;
+}
+
+.button:hover {
+  background: #a1a1a1;
+  text-decoration: none;
+}
+
+
+.heroImage .button {
```

Comment:
I think here, instead of using CSS specificity to override the default grey button and make it yellow, it'd be better to use two classes on this particular button and let this second class override the first. But really, that's a matter of preference 😄